step2_spritesheet_to_generative_sheet/Solana/metaplex.js

```diff
@@ -10,6 +10,8 @@ const {
   NFTName,
   collectionName,
   collectionFamily,
+  NFTPrefix,
+  name_prefix_gap,
```

Comment:
I think you need to fetch the code from main branch, this branch looks stale. Also underscores

Reply:
actually wanna just fix it from your end? im new to github and all this terminology is causing lots of headaches XD. It would probably be much more convenient if you did it ;-;

Reply:
ok that ^ looks like im kinda annoyed, im not btw; I really just...suck 😂

Reply:
Yea no problem! i'll fix it haha

Reply:
no worries

Reply:
thanks 😅